client/utils/utils.go

```diff
@@ -163,12 +163,14 @@ func PrintUnsignedStdTx(
 // SignStdTx appends a signature to a StdTx and returns a copy of a it. If appendSig
```

Comment:
s/a copy of a it/a copy of it/

Would you mind to fix this comment please?

Reply:
Done.